_docs/datasources/google ads.md

```diff
@@ -25,29 +25,44 @@ After authorization you need to set the data source parameters.
 
 For a full picture of advertising costs, you need to add all your active advertising accounts (2) (see the "How to Find Your Account ID" section to find out where to find the advertising account ID).
 
-**Import "Keyword Performance" report** (3) - allows you to import statistics on costs, without normalizing the values ​​in the table.
+**Import "Keyword Performance" report** (3) - allows you to import "Keyword Performance" report, without normalizing the values ​​in the table.
```

Comment:
@bogomolov89 maybe rewrite to:
**Import "Keyword Performance" report** (3) - enable import "Keyword Performance" report

Reply:
I think it's fine as it is, but it could also be:
Import "Keyword Performance" report (3) - enable "Keyword Performance" report import, without normalizing the values ​​in the table.